packages/stryker/src/transpiler/CoverageInstrumenterTranspiler.ts

```diff
@@ -8,19 +8,19 @@ import { FileCoverageData, Range } from 'istanbul-lib-coverage';
 
 const COVERAGE_CURRENT_TEST_VARIABLE_NAME = '__strykerCoverageCurrentTest__';
 
-export interface FileCoverageMaps {
+export interface CoverageMaps {
   statementMap: { [key: string]: Range };
-  fnMap: { [key: string]: { loc: Range } };
+  fnMap: { [key: string]: Range };
 }
 
-export interface FileCoverageDataDictionary {
-  [file: string]: FileCoverageMaps;
+export interface CoverageMapsByFile {
+  [file: string]: CoverageMaps;
```

Comment:
Why don't we store the filename in the coveragemap?

Reply:
As a sneak preview, i'm refactoring it like this:

```
abstract class CoveredStatement {
  fileName: string;
  range: Range;
  abstract getCoveringTestIds(): number[];
}
```

Probably a lot cleaner like this.